export/app.yaml

```diff
@@ -4,6 +4,8 @@ threadsafe: true
 service: cloud-datastore-admin
 
 libraries:
+- name: requests
```

Comment:
This won't work either (in fact I get a "the library requests is not supported" error). You can only use this approach for the libs mentioned here: https://cloud.google.com/appengine/docs/standard/python/tools/built-in-libraries-27. For others, they'd need to be imported in an analogous fashion to how we do it in the main app...

Reply:
I'm thinking we should just split this into its own repository, as you've suggested before. In that repo, let's make this a Python 3 app and use that system to set things up.

WDYT?

Reply:
Yup no objections!